packages/components/src/Tooltip/Tooltip.test.tsx

```diff
@@ -13,8 +13,11 @@ test("tooltip shouldn't show up", () => {
       <div>{content}</div>
     </Tooltip>,
   );
-  expect(queryByText(message)).toBeNull();
-  expect(getByText(content)).toBeTruthy();
+
+  await waitFor(() => {
+    expect(queryByText(message)).not.toBeInTheDocument();
+  });
+  expect(getByText(content)).toBeInTheDocument();
```

Comment:
You shouldn't need to await `queryByText` as it doesn't require async, see [React Testing Library Cheatsheet](https://testing-library.com/docs/react-testing-library/cheatsheet/)

Reply:
I'm actually waiting for the `.not.toBeInTheDocument();` bit as that one takes a moment to appear

Reply:
actually wait. the way the waitFor works is it would retry for a bit and it it still fails, then it will fail. 

> waitFor may run the callback a number of times until the timeout is reached

Reply:
you shouldn't need to do that though, look at this example: https://testing-library.com/docs/guide-disappearance/#nottobeinthedocument

Reply:
@joelwarrington-jobber in most cases, yup. until you deal with things that needs to transition out in which case there's a bit of delay before that element truly disappears